app/src/organisms/ChooseRobotSlideout/FileCard.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { css } from 'styled-components'
+import {
+  ALIGN_CENTER,
+  BORDERS,
+  Btn,
+  COLORS,
+  DIRECTION_COLUMN,
+  Flex,
+  Icon,
+  JUSTIFY_SPACE_BETWEEN,
+  SPACING,
+  StyledText,
+} from '@opentrons/components'
+import type { CsvFileParameter, RunTimeParameter } from '@opentrons/shared-data'
+
+interface FileCardProps {
+  error: string | null
+  fileRunTimeParameter: CsvFileParameter
+  runTimeParametersOverrides: RunTimeParameter[]
+  setRunTimeParametersOverrides?: (_: RunTimeParameter[]) => void
+}
+
+export const FileCard = (props: FileCardProps): JSX.Element => {
+  const {
+    error,
+    fileRunTimeParameter,
+    runTimeParametersOverrides,
+    setRunTimeParametersOverrides,
+  } = props
+
+  return (
+    <Flex
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing4}
+      width="100%"
+    >
+      <Flex
+        height="2.75rem"
+        padding={`${SPACING.spacing8} ${SPACING.spacing12}`}
```

Comment:
```suggestion
        padding={SPACING.spacing8}
```

<img width="576" alt="Screenshot 2024-06-13 at 3 22 25 PM" src="https://github.com/Opentrons/opentrons/assets/474225/8a368037-0f2b-4ba4-b902-cef578805519">